src/payment/strategies/ppsdk/ppsdk-payment-processor.ts

```diff
@@ -0,0 +1,18 @@
+import { OrderPaymentRequestBody } from '../../../order';
+import { PPSDKPaymentMethod } from '../../ppsdk-payment-method';
+
+import { initializeNone, isNone } from './initializers';
+
+export type PaymentProcessor = (payment: OrderPaymentRequestBody | undefined) => Promise<void>;
+
+type GetPaymentProcessor = (paymentMethod: PPSDKPaymentMethod) => PaymentProcessor | undefined;
+
+export const getPaymentProcessor: GetPaymentProcessor = paymentMethod => {
```

Comment:
It seems to me that this is just another resolver, which mean you can probably just use the existing `Registry` class to achieve what you want instead of creating another concrete implementation. e.g.: You can create a registry of all the PPSDK processors/handlers and resolve them based on their PPSDK strategy type.

Reply:
Done: https://github.com/bigcommerce/checkout-sdk-js/pull/1150/files#diff-6df6aa7ed80a40f81baf653e8cd4c02262704158ca6dc151b357c4fdd4db8a39R8